tfx/experimental/distributed_inference/graphdef_experiments/subgraph_partitioning/beam_pipeline.py

```diff
@@ -0,0 +1,220 @@
+# Copyright 2020 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Run batch inference in beam on partitioned graphs.
+
+After getting execution_specs from graph partitioning, we'd like to arrange,
+load, and execute the subgraphs in beam.
+
+Execution_specs:
+    <--closer to inputs                   closer to outputs-->
+        --------------------------------------------------
+        |          |          |          |               |
+        |  spec 1  |  spec 2  |  spec 3  | ......        |
+        |          |          |          |               |
+        --------------------------------------------------
+                 /              \
+                /                \
+               / ---------------- \
+                 |   Subgraph   |
+                 ----------------
+                 |    Inputs    |
+                 ----------------
+                 |   Outputs    |
+                 ----------------
+                 | is_remote_op |
+                 ----------------
+"""
+
+import tensorflow as tf
+import apache_beam as beam
+import copy
+
+
+@beam.ptransform_fn
+def ExecuteOneGraph(pcoll,
+                    op_to_execution_specs,
+                    op_to_remote_op_name_mapping,   # This information was stored in py_func's feed_dict. We don't have it.
```

Comment:
Max line length should be 80 chars: https://google.github.io/styleguide/pyguide.html#32-line-length
Indentation is 2 chars, not 4.

Check out the tensorflow coding style: https://www.tensorflow.org/community/contribute/code_style

Run pylint and fix lint errors: https://www.pylint.org/

Reply:
Corrected line length, indentation, and lint errors.